internal/monitor/controller.go

```diff
@@ -132,6 +132,14 @@ func (cm *PodMonitorType) controllerModePodHandler(pod *v1.Pod, eventType watch.
 				pod.ObjectMeta.Namespace, pod.ObjectMeta.Name, pod.Spec.NodeName, initialized, ready, taintnosched, taintnoexec, taintpodmon)
 			// TODO: option for taintnosched vs. taintnoexec
```

Comment:
Please remove this TODO. It seems that this is no longer required.

Reply:
will do.